src/dbnode/generated/proto/namespace/schema.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package namespace;
+
+// SchemaMeta contains schema metadata.
+message SchemaMeta {
+    // message name as it appears in proto file descriptor.
+    string messageName = 1;
+}
+
+// SchemaOptions contains schema options for a namespace.
+message SchemaOptions {
+    // repo contains a history of deployed schema defintions in form of proto file descriptors.
+    FileDescriptorRepo repo = 1;
+    // schemas contains a set of named schemas for a namespace.
+    // a namespace can have multiple schemas.
+    // series identifies its schema by schema name (the map key) and a version (repo version).
```

Comment:
I think the last sentence might be a little more clear if you leave out any references to a "series" and just say: "A schema is uniquely identified by its name (the map key) and a version number (the repo version)."

Reply:
sounds good.

Reply:
on a second thought, it is more for series level metadata, schema definition is one of them (schema definition references the proto message name in the latest file descriptor repo).